Cargo.toml

```diff
@@ -19,11 +19,13 @@ showata = {version = "0.2.0", optional = true}
 anyhow = {version = "1.0.22", optional = true }
 ndarray = {version = "0.13", features = ["serde"], optional = true}
 csv = {version = "1.1", optional = true}
+rulinalg = {version ="0.4.2", optional=true}
+nalgebra = {version ="0.19.0", features = ["serde-serialize"], optional=true}
 
 [dev-dependencies]
 csv = "1.1"
 ndarray-rand = "0.11"
 
 [features]
 show_vega = ["showata", "anyhow"]
-default = ["show_vega", "csv", "ndarray"]
+default = ["show_vega", "csv", "ndarray", "nalgebra", "rulinalg"]
```

Comment:
I'm not sure we want to have `ndarray`, `nalgebra` and `rulinalg` by default. We should either pick one or make all three opt-ins

Reply:
Also, features should be explained in project `README.md`

Reply:
I agree. Currently only `ndarray` and `nalgebra` are in active development so I think we can make it one of it as default or all three as opt-ins.
I will push a updated project `README.md` the one in example is already updated.

Reply:
Fixed by daadf47 :)